ft/onto/base_ontology.py

```diff
@@ -311,7 +322,12 @@ class PredicateLink(Link):
     ParentType = PredicateMention
     ChildType = PredicateArgument
 
-    def __init__(self, pack: DataPack, parent: Optional[Entry] = None, child: Optional[Entry] = None):
```

Comment:
I guess we don't need to `black` this file. The only update here should be the new `Character` class.